client/lib/post-normalizer/index.js

```diff
@@ -604,7 +604,11 @@ normalizePost.content = {
 			'videopress.com',
 			'vimeo.com',
 			'cloudup.com',
-			'soundcloud.com'
+			'soundcloud.com',
+			'rdio.com',
+			'rd.io',
```

Comment:
Should we skip rdio since <a href="http://variety.com/2015/digital/news/rdio-shutdown-shutting-down-free-plan-1201646753/">they're shutting down</a>?